version.properties

```diff
@@ -0,0 +1 @@
+version=2.0.1-SNAPSHOT
```

Comment:
I think we should have version=2.0.1 in version.properties, then let build.gradle to add up the `-SNAPSHOT` if isSnapshot=false.
Else, according to your code you do not have action to remove `-SNAPSHOT` for official release, except manually changing the version.properties.

Reply:
👍 that makes sense, changed it